cli/pom.xml

```diff
@@ -42,6 +42,11 @@
             <groupId>info.picocli</groupId>
             <artifactId>picocli-shell-jline3</artifactId>
         </dependency>
+        <dependency>
```

Comment:
This is not a new dependency. It is listed in the parent POM already. The issue is if you do `assertTrue(String.contains())` you only get a boolean `true` or `false` and no contextual information about what failed. When a test fails, you don't really know why. The AssertJ method gives you both the expected and actual result, maybe even the position where it failed (unless that's just a a Hamcrest feature).

Reply:
I guess this is personal preference. If the tests are passing, then its just a nicety.

Reply:
An example failure as it is now is `AppTest.testHelp:26 expected: <true> but was: <false>`. But it's comparing `String`s, so in terms of seeing why it failed. Seeing the strings would help. The normal `assertEquals` does this, but doesn't work for substrings.

I believe it's in Hamcrest, not AssertJ, but it would actually show something like `expected: "foo bar" but was "foo b[ox]"`. Now, instead of true/false you see the exact reason and location it failed.